tests/test_storage.py

```diff
@@ -0,0 +1,128 @@
+import os
+
+import boto3
+import pytest
+from moto import mock_aws
+
+from polaris.hub.storage import S3Store
+
+
+@pytest.fixture(scope="function")
+def aws_credentials():
+    """
+    Mocked AWS Credentials for moto.
+    """
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
+
+
+@pytest.fixture(scope="function")
+def mocked_aws(aws_credentials):
+    """
+    Mock all AWS interactions
+    Requires you to create your own boto3 clients
+    """
+    with mock_aws():
+        yield
+
+
+@pytest.fixture
+def s3_store(mocked_aws):
+    # Setup mock S3 environment
+    s3 = boto3.client("s3", region_name="us-east-1")
+    bucket_name = "test-bucket"
+    s3.create_bucket(Bucket=bucket_name)
+
+    # Create an instance of your S3Store
+    store = S3Store(
+        path=f"{bucket_name}/prefix",
+        access_key="fake-access-key",
+        secret_key="fake-secret-key",
+        token="fake-token",
+        endpoint_url="https://s3.amazonaws.com",
```

Comment:
**Question:** Is `aws_credentials()` needed if you hard-code the same variables here?

Reply:
AWS-related libraries have a cascade algorithm when looking for credentials, where they look in ~3-4 places for them. Having `aws_credentials` means there will always be a dummy value if we forget to explicitly set something somewhere in the tests.